versions/1/data_stream/manifest.spec.yml

```diff
@@ -1,37 +1,37 @@
 ##
-## Describes the specification for a dataset's manifest.yml file
+## Describes the specification for a data stream's manifest.yml file
 ##
 spec:
   # Everything under here follows JSON schema (https://json-schema.org/), written as YAML for readability
   type: object
   additionalProperties: false
   properties:
     name:
```

Comment:
This would have to become `dataset` as the `dataset.name` property was renamed to `data_stream.dataset`.

Reply:
Fixed in 383c692.